src/Umbraco.Web/HtmlStringUtilities.cs

```diff
@@ -20,9 +20,25 @@ public sealed class HtmlStringUtilities
         /// </summary>
         /// <param name="text">The text.</param>
         /// <returns>The text with text line breaks replaced with HTML line breaks (<br/>)</returns>
+        [Obsolete("Doesn't html encode the text. Use ReplaceLineBreaks instead")]
         public HtmlString ReplaceLineBreaksForHtml(string text)
         {
-            return new HtmlString(text.Replace("\r\n", @"<br />").Replace("\n", @"<br />").Replace("\r", @"<br />"));
+            return new HtmlString(text.Replace("\r\n", @"<br />").Replace("\n", @"<br />").Replace("\r", @"<br />"));            
+        }
+        
+        /// <summary>
+        /// Html encodes the text and replaces text line breaks with HTML line breaks.
+        /// </summary>
+        /// <param name="text">The text.</param>
+        /// <returns>The html encoded text with text line breaks replaced with HTML line breaks (<br/>)</returns>
+        public HtmlString ReplaceLineBreaks(string text)
+        {
+            var value = HttpUtility.HtmlEncode(text ?? string.Empty)
```

Comment:
`HttpUtility.HtmlEncode()` and `HtmlString` happily accept `null` values, so no need to create empty strings. This also ensures the `Replace` calls are skipped when the text is `null` (by using the null-conditional operator):

```suggestion
            var value = HttpUtility.HtmlEncode(text)?
```

Reply:
Sure, makes sense 👍